app/views/products/index.html.haml

```diff
@@ -22,4 +22,4 @@
             .form-group
               %button.btn.btn-success{ ng: { click:"addToCart(product)" }}
                 Add To Cart
-              %input.input-sm{ type: "number", ng: { model: "product.quantity" }}
\ No newline at end of file
+              = select_tag('', options_for_select((1..10)), {'ng-model' => "product.quantity", 'ng-init' => '' })
```

Comment:
Whats with the empty ng-init again?  

Are you limiting the customer's add to cart quantity because of a secret business plan or because you have an issue with displaying inventory quantity?